brainrender/_colors.py

```diff
@@ -0,0 +1,115 @@
+import matplotlib.cm as cm_mpl
+import numpy as np
+from vedo.colors import getColor
+from vedo.colors import colors as vcolors
+import random
+
+
+_mapscales = cm_mpl
+
+
+# Some nice colors
+red = "#fc330f"
```

Comment:
most `vedo` functions will accept rgb colors and support matplotlib's color names.
It's not explicitly referenced in the docstrings because hex or color names are easier to handle.